src/Azure.DataApiBuilder.Core/Utils.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using System.Reflection;
+
+namespace Azure.DataApiBuilder.Core
```

Comment:
Since this is new, let's go with a top-level namespace:

```suggestion
namespace Azure.DataApiBuilder.Core;
```

You'll need to remove the namespaces `{}` and then drop the indentation.

Reply:
Fixed.